pkg/volume/fc/fc.go

```diff
@@ -235,11 +235,56 @@ func (plugin *fcPlugin) newUnmapperInternal(volName string, podUID types.UID, ma
 }
 
 func (plugin *fcPlugin) ConstructVolumeSpec(volumeName, mountPath string) (*volume.Spec, error) {
```

Comment:
can you add a unit test too?

Reply:
@rootfs 
Do you have an idea to mock `mount.GetMountRefs(mounter, mountPath)`?

Reply:
done.